docs/tutorials/4 Request Handling.md

```diff
@@ -57,6 +57,12 @@ FHIR® bundle requests have some advantages over direct requests. The number of
 
 #### Step 4.2.1: Declaration of Groups
 Depending on the use case, every binding can be configured to use either `batch` or `transaction`. This can be done by introducing binding groups. Application developers decide which binding groups are necessary for a particular UI5 application and which request type should be used for these groups before assigning these groups to the associated bindings. With this concept, it’s possible to reduce the number of HTTP requests to an appropriate level for the respective UI5 application. The declaration of groups is done in the `manifest.json` of the UI5 application.
+The `fullUrlType` for batch or transaction can be configured. Supported Types are `uuid` and `absolute`. 
```

Comment:
```suggestion
With using `batch` or `transaction` as request type, the different requests are packed in a bundle as bundle entries. Every bundle entry has to contain the property `fullUrl`. This `fullUrl` property is of FHIR® datatype [uri](http://hl7.org/fhir/datatypes.html#uri) which can be an `url`, `canonical`, `uuid` or `oid`. Since version 1.1.0 `url` and `uuid` is supported by openui5-fhir. With `uuid` the `fullUrl` property is generated with following syntax `urn:uuid:uuidv4`, e.g. "urn:uuid:e91a94c7-b874-4b46-af6a-3412b487c3af". With `url` the `fullUrl` property is generated with following syntax `http://fhirServerUrl/ResourcePath`, e.g. "http://example.com/Patient/123". 
```

Reply:
done